js/drafts/composer.js

```diff
@@ -155,8 +155,15 @@ function Composer(newRecords, account, identity) {
 
   body.attachments.forEach(function(attachment) {
     try {
-      var attachmentNode = new MimeNode(attachment.type,
-                                        { filename: attachment.name });
+      var attachmentNode = new MimeNode(
+        attachment.type,
+        {
+          // This implies Content-Disposition: attachment
+          filename: attachment.name
+        });
+      // Explicitly specify base64 encoding; this is the default for non
+      // text/* parts, but it's important for us to not break on those, too.
+      attachmentNode.setHeader('Content-Transfer-Encoding', 'base64');
```

Comment:
I had a bit of trouble following the comment. I first read it as text/* don't have base64 encoding, and it implied an if/else case, but I think what is meant is that by default all attachments are base64 encoded. For text ones, this is not so much of an issue if we forget to do this, but best to be very specific so that all cases are handled uniformly?